_includes/desktop-table.html

```diff
@@ -1,145 +1,122 @@
+		  {%- else -%}
+			{%- assign progress_tweet = page.progress_tweet -%}
+			{%- assign workonit_tweet = page.workonit_tweet -%}
+		  {%- endif -%}
+		  <tr class="desktop-tr{% if website.bch == false %} no-bch{% endif %}">
+			{%- if website.status -%}
+			  <td class="main progress">
+				<div class="title">
+{% include segment/row-logo.html section=section_id img=website.img site=website.name %}
+				  <a class="name" href="{{ website.url }}" target="_blank">{{ website.name }}</a>
+{%- include exception.html website=website -%}
+				</div>
+				<div class="progress-info">
+				  <a class="fluid ui mini orange button" href="{{website.status}}" target="_blank">
+					In Progress!
+				  </a>
+				</div>
+			  </td>
+			  <td class="twitter progress" colspan="4">
+				<a data-tooltip="{{page.link_tweet|replace:'TWITTERHANDLE',website.twitter}}" data-position="top center" data-inverted="" class="ui twitter mini button" href="https://twitter.com/share?url={{site.url|cgi_escape}}&amp;text={{progress_tweet|replace:'TWITTERHANDLE',website.twitter|cgi_escape}}&amp;hashtags={{page.hash|cgi_escape}}" target="_blank"><i class="twitter big icon"></i> {{page.link_progress}}</a>
+			  </td>
+			{%- elsif website.bch or website.btc -%}
+			  <td class="main {% if website.bch %}positive{% else %}negative{% endif %}">
+				<div class="title">
+				  {% include segment/row-logo.html section=section_id img=website.img site=website.name %}
+				  <a class="name" href="{{ website.url }}" target="_blank">{{ website.name }}</a>
+				  {%- include exception.html website=website -%}
+				  {%- if website.twitter and website.bch -%}<a class="social-name" href="https://twitter.com/{{ website.twitter }}" target="_blank" data-tooltip="@{{ website.twitter }}" data-position="top center" data-inverted=""><i class="twitter icon"></i></a>{%- endif -%}
+				  {%- if website.facebook and website.bch -%}<a class="social-name" href="https://facebook.com/{{ website.facebook }}" target="_blank" data-tooltip="fb.com/{{ website.facebook }}" data-position="top center" data-inverted=""><i class="facebook f icon"></i></a>{%- endif -%}
+				</div>
+			  </td>
+			  <td class="{% if website.bch %}positive{% else %}negative{% endif %} icon">
+				{%- if website.bch -%}
+				<div class="coin-wrap green-coin-wrap" data-tooltip="Bitcoin Cash accepted at {{ website.name }}" data-position="top center" data-inverted="">
+				  <img src="/img/coin-bch.png" alt="Bitcoin Cash">
+				</div>
+				{%- else -%}
+				{%- if website.twitter -%} <a data-tooltip="{{page.link_tweet|replace:'TWITTERHANDLE',website.twitter}}" data-position="right center" data-inverted="" class="ui twitter mini button" href="https://twitter.com/share?url={{site.url|cgi_escape}}&amp;text={{workonit_tweet|replace:'TWITTERHANDLE',website.twitter|cgi_escape}}&amp;hashtags={{page.hash|cgi_escape}}" target="_blank"><i class="twitter big icon"></i></a>{%- endif -%}
```

Comment:
@t0p0lin0 this and line 104 are virtually identical but this one has `data-position="right center"` but the other has `data-positon="left center"`

could they both be the same? would love to move these things out to segment includes and/or re-arrange things so we dont have to define them twice similar to what I did with the `listings.html`

Reply:
I've changed it and `left center` to `right center` and everything seems to be looking fine. I'll leave the final tweaks/moves to you.